package.json

```
@@ -41,9 +41,8 @@
   },
   "homepage": "https://github.com/developit/preact-redux",
   "peerDependencies": {
-    "preact": "<10",
-    "redux": ">=2",
-    "preact-context": ">=1.1.0"
+    "preact": ">10",
```

IIRC needs to be inclusive?

```suggestion
    "preact": ">=10",
```